### ISSUE TITLE
Blocking pornographic content

### ISSUE THREAD

**Issue by cruxoooo** [nextdns/metadata#1205]
Hello,

The pornographic filter has been lacking since switching to internal development. I would expect the stevenblack porn hosts list to be the baseline: https://github.com/StevenBlack/hosts but this is sadly not the case. Any chance of adding websites from his hosts/porn to your filter?
There is also a lack of "adult content/smut" filtering. Any chance of adding such a filter category in the feature?

Thank you.

**Comment by BourbonCrow** [comment]
i agree the with that the internalized filters should be better maintained or atleast open source here on github so ppl can report to it also stevenblacks as a baseline is prollyy a good idea his lists are amazing
